### ISSUE TITLE
Vectors in unstructered grid

### ISSUE THREAD

**Issue by m3m0ry** [jipolanco/WriteVTK.jl#37]
Hi,
i am struggling to get vectors going.
This is my codebase: [here](https://github.com/m3m0ry/gravitation)
```
    cells = [MeshCell(VTKCellTypes.VTK_VERTEX, [i]) for i = 1:N]
    outfile = vtk_grid(string("my_vtk_file",i), points.position, cells) do vtk 
        vtk_cell_data(vtk, points.mass, "Mass")
        vtk_cell_data(vtk, points.velocity, "Velocity")
        vtk_point_data(vtk, points.mass, "Pointmass")
        vtk_point_data(vtk, points.velocity, "pointvelocity")
    end 
```
To keep things simple i am in 2D.
i have some set of points, each point has a velocity and a mass.
so i create an unstructured grid with the point's positions and an array of MeshCells of the type Vertex.
Then i add some data to it. (I do both, point_data and cell_data, in order find out my mistake)
The 'Mass' and 'Pointmass' are working just fine. (scalar data works).

The Vector data is added as well. Everything seems fine.
However in paraview i cannot select it.
If i try to put a simple Arrow-Glyph on that, i cannot choose its Vectors in the Active Attribute part.

I thought it is an paraview bug, so i tried their python interface, where you can directly specify the vector, however that does not work either.
I am still suspecting paraview for this behavior, however i am also still not sure if i use the vectors correctly with WriteVTK.

**Comment by m3m0ry** [comment]
I have now tested it on a different computer with a different paraview version and still it does not work.

I am still thinking that the error is either with paraview, or with the object 40-60 cm away from the screen.

**Comment by jipolanco** [comment]
Hi, I quickly tested your gravitation.jl code and I find the same issue when trying to create an arrow glyph on paraview 5.4.1. I'd say the problem is that paraview expects a 3-component vector in order to draw an arrow. Can you try adding a third dimension to your `velocity` vector, setting the z-components of the velocity to zero? You shouldn't need to do this for the position, since that's handled automatically by WriteVTK.

**Comment by m3m0ry** [comment]
Yep, that was it.

Thank you!!!


TLDR;
Paraview expects vectors with 3 components.
